mos-platform/geos-cbm/clang.cfg

```diff
@@ -0,0 +1,5 @@
+-mllvm -inline-threshold=0
+-T vlir.ld
+-mlto-zp=107
+-Wl,-linit-stack,--no-check-sections
```

Comment:
Something is fishy about `-linit-stack`; merging `common-init-stack` into `libcrt0` `CMakeLists.txt` should ensure that it's always present on the link line without any further intervention. Does the example build without this? If not, what's the error message?